content/swagger/resources/attributes/definitions/attribute_option.yaml

```diff
@@ -1,14 +1,17 @@
 type: object
+required: ["code"]
 properties:
   code:
     type: string
     description: Code of option
   attribute:
     type: string
     description: Code of attribute related to the attribute option
+    x-validation-rules: It is the same attribute code that the one given as path parameter
```

Comment:
it is the same attribute code `than`

Reply:
no, `as`

https://forum.wordreference.com/threads/same-than-same-as-same-that.400069/

Reply:
I was not talking about the `as` but the part `It is the same attribute code that` I don't know if the `that` is correct. I check on several websites ;).

Reply:
You didn't check my link. 
`It is the same attribute code as the one given `